src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -8,10 +8,40 @@
 // @grant       none
 // ==/UserScript==
 /* global $ */
+    var targetNode = document.querySelector('.listusersrow, .listusersrow table tr td');
+    if (targetNode) {
+      var observerOptions = {
+        childList: true,
+        subtree: true
+      };
 
+      var observer = new MutationObserver(function(mutationsList) {
+        routine();
+      });
+
+      observer.observe(targetNode, observerOptions);
+    } else {
+      //console.error("Target node with class 'listusersrow' or 'listusersrow table tr td' not found!");
+    }
 /*
  * Routine function of the script.
  */
+var listenerRunning = false;
+
+document.addEventListener("click", function(event) {
+if (!listenerRunning) {
+listenerRunning = true;
+    try
+    {
+     if (event.target.matches('.fastattack')) {
+      routine();
+    }
+    }catch{}
```

Comment:
catch without any log whatsoever sounds a bit suspicious. what errors does it catch in practice? is the catch really necessary?

Reply:
ah, i forgot the try and catch to remove, added this for testing ;)

removed